File: web/src/data/historyData.ts
// Local demo data for the History screen. Mirrors the reference.

export type HistoryStatus = 'on-time' | 'late' | 'missed'

export interface HistoryRow {
  id: string
  time: string
  name: string
  detail: string
  status: HistoryStatus
  badge: string // "On Time" | "15m Late" | "Missed"
}

export interface HistoryGroup {
  label: string
  date: string
  rows: HistoryRow[]
}

export const historySummary = {
  period: 'This Week',
  percent: 0,
  taken: 0,
  late: 0,
  missed: 0,
  dosesText: '0 of 0 doses taken',
}

export const historyGroups: HistoryGroup[] = []

File: ios/MediCenter/Features/History/HistoryData.swift
import SwiftUI

// Local demo data for the History screen. Mirrors the web app.

enum HistoryStatus { case onTime, late, missed }

struct HistoryRow: Identifiable {
    let id = UUID()
    let time: String
    let name: String
    let detail: String
    let status: HistoryStatus
    let badge: String
}

struct HistoryGroup: Identifiable {
    let id = UUID()
    let label: String
    let date: String
    let rows: [HistoryRow]
}

enum HistoryData {
    static let summary = (period: "This Week", percent: 0, taken: 0, late: 0, missed: 0, doses: "0 of 0 doses taken")

    static let groups: [HistoryGroup] = []
}
